### PR TITLE
Reforgestones cleanup

### PR DESCRIPTION
**Pull request by Thunderblade73**
mostly formatting fixes for the reforgeAbility.
Removed Luxurious Spool, since this item does not exit anymore.
Added Boo Stone.
Changed ItemType of Presumend Gallon of Red Paint to Cloak

### CODE REVIEW

File: constants/reforgestones.json [hunk 1]
```diff
@@ -290,6 +290,30 @@
...
+    "itemTypes": "ARMOR",
+    "requiredRarities": [
+      "EPIC",
+      "LEGENDARY"
+    ],
+    "reforgeCosts": {
+      "EPIC": 10000,
+      "LEGENDARY": 10000
+    },
+    "reforgeAbility": {
```
Comment (jani270) [inline constants/reforgestones.json:306]: You could just do `"reforgeAbility": "Grants §a+1 §5Fear§7.",` here.

Comment (Thunderblade73) [reply constants/reforgestones.json]: thx. And done.

File: constants/reforgestones.json [hunk 2]
```diff
@@ -290,6 +290,27 @@
...
+    "requiredRarities": [
+      "EPIC",
+      "LEGENDARY"
+    ],
+    "reforgeCosts": {
+      "EPIC": 10000,
+      "LEGENDARY": 10000
+    },
+    "reforgeAbility": "Grants §a+1 §5Fear§7.",
+    "reforgeStats": {
```
Comment (jani270) [inline constants/reforgestones.json:307]: Please remove reforgeStats as its empty

**Review by jani270 — CHANGES_REQUESTED** [review]
Sorry didnt see it the first time

File: constants/reforgestones.json (hunk 2 above)
Comment (Thunderblade73) [reply constants/reforgestones.json]: Was unsure if it worked without reforgeStats. But tested it now and nothing bad happend so it should be fine without it.

**Review by jani270 — APPROVED** [review]

**Pull request merged**